build/version.go

```diff
@@ -43,7 +43,7 @@ const (
 	AppMinor uint = 16
 
 	// AppPatch defines the application patch for this binary.
-	AppPatch uint = 1
+	AppPatch uint = 2
```

Comment:
Does this need to be updated anywhere else?

Reply:
Nope, everything else (cli, etc) will read this value.